azure-cli/ad/deploy.swagger.ps1

```diff
@@ -0,0 +1,80 @@
+param (
```

Comment:
Rewrite this to a function instead of a script

Reply:
Why should this be a function when all other deploy files are scripts?

Reply:
We have already rewritten a lot of the deploy.*.ps1 files to functions, and will continue doing so, so it's not really **all other**.
Functions benefit of being able to return values that can be passed to other functions (see [Provision-AppServicePlan](https://github.com/atc-net/atc-snippets/blob/main/azure-cli/appservice/Provision-AppServicePlan.ps1)).
Also, with functions, you get the ability to hit F12 in VSCode and it will take you to the implementation and it looks less hacky.

Reply:
I will change to a function. Correct me if I am wrong but it seems you are also trying to change the structure of the files so that they start with words like Get/Provision/Update? Should I also rename this file to Provision-SwaggerSPN?